src/utils/fetch-bounties.ts

```diff
@@ -32,6 +33,20 @@ export async function fetchBountiesAndChildBounties(showProgress = true) {
 			childBounties.sort((a, b) => (a.id > b.id ? -1 : 1));
 		});
 
+		await fetchAllProxies();
+
+		bounties.forEach((bounty) => {
+			if (bounty.curator) {
+				bounty.curatorMultisigAccount = get(proxies)?.get(bounty.curator);
```

Comment:
Could the map of proxies be directly returned by `fetchAllProxies`?

Reply:
[732a285](https://github.com/galaniprojects/Polkadot-Bounty-Manager/pull/491/commits/732a2850f3ac8217422091fdeaef382b990d8aa8)